units/meson.build

```diff
@@ -180,7 +180,7 @@ in_units = [
          'sysinit.target.wants/'],
         ['systemd-importd.service',              'ENABLE_IMPORTD',
          'dbus-org.freedesktop.import1.service'],
-        ['systemd-initctl.service',               ''],
+        ['systemd-initctl.service',              'HAVE_SYSV_COMPAT'],
```

Comment:
Please also update `man/systemd-initctl.service.xml`.

Also, please update `executable('systemd-initctl')` like the above.

Reply:
xml is fixed